integrations/os2sync/os2mo.py

```diff
@@ -251,9 +254,49 @@ def kle_to_orgunit(orgunit, kle):
         orgunit["ContactForTasks"] = list(sorted(contactfortasks))
 
 
+def is_ignored(unit, settings):
+    """Determine if unit should be left out of transfer
+
+    Example:
+        >>> settings={
+        ... "OS2SYNC_IGNORED_UNIT_LEVELS": ["10","2"],
+        ... "OS2SYNC_IGNORED_UNIT_TYPES":['6','7']}
+        >>> unit={"org_unit_level":{"uuid":"1"}, "org_unit_type":{"uuid":"5"}}
+        >>> is_ignored(unit, settings)
+        False
+        >>> unit={"org_unit_level":{"uuid":"2"}, "org_unit_type":{"uuid":"5"}}
+        >>> is_ignored(unit, settings)
+        True
+        >>> unit={"org_unit_level":{"uuid":"1"}, "org_unit_type":{"uuid":"6"}}
+        >>> is_ignored(unit, settings)
+        True
+
+    Args:
+        unit: The organization unit to enrich with kle information.
+        settings: a dictionary
+
+    Returns:
+        Boolean
+    """
+
+    return (
+        unit.get("org_unit_level") and unit["org_unit_level"]["uuid"] in settings[
+            "OS2SYNC_IGNORED_UNIT_LEVELS"
+        ]
+    ) or (
+        unit.get("org_unit_type") and unit["org_unit_type"]["uuid"] in settings[
+            "OS2SYNC_IGNORED_UNIT_TYPES"
+        ]
+    )
```

Comment:
For at reducere duplikeret kode, ville jeg nok lave en meta version med dokumentation og tests, og så laver en LoraCache og MO udgave, der kalder med deres data respektivt. - Alternativt kunne LoraCache / MO løftes til et højere nivuea, så de får fælles interface og det er på konstruktion time at det afgøres hvilken der benyttes.